perun-wui-registrar/src/main/java/cz/metacentrum/perun/wui/registrar/widgets/items/ListBox.java

```diff
@@ -95,13 +96,42 @@ protected void setValidationTriggers() {
 		if (isOnlyPreview()) {
 			return;
 		}
-		for (ExtendedTextBox input : inputList) {
-			input.addBlurHandler(new BlurHandler() {
+		if (this.validator instanceof SshKeysListBoxValidator) {
```

Comment:
Please do not rely on validator instance, since this method is called in super constructor when this specific validator is not yet set (as it happens later in local/child constructor).
Right now content of form item is not actually validated against API until you add/remove one of the list items, when the widget get re-generated from scratch. I suggest to use "destination attribute" name to determine how it should be validated (similar to the code in constructor). I think same apply to the TextArea and TextField form items.